packages/kuma-gui/src/app/service-mesh/debug.ts

```diff
@@ -0,0 +1,21 @@
+import { XCopyButtonDebug } from '@/app/x'
+import type { ServiceDefinition, Token } from '@/services/utils'
+import { token } from '@/services/utils'
+
+export const services = (app: Record<string, Token>): ServiceDefinition[] => [
+  [token('development.components'), {
+    service: () => {
+      return [
+        (name: string, item: unknown) => {
+          if (name === 'XCopyButton') {
+            return [name, XCopyButtonDebug]
+          }
+          return [name, item]
+        },
+      ]
+    },
+    labels: [
+      app.componentWalkers,
+    ],
+  }],
+]
```

Comment:
We choose to add the `XCopyButtonDebug` for our application, other users of `x` might not want to do this, and we don't want to provide a debug container in `x` to keep it dependency free, so I moved this to `service-mesh`.

This file only gets included during development, I also checked a production build to make sure that `XCopyButtonDebug` isn't compiled into a prod build i.e. `XCopyButtonDebug` is tree-shaken out of `x` when its not imported anywhere.